index.d.ts

```diff
@@ -1,14 +1,13 @@
 import { FastifyPluginCallback } from 'fastify';
 import { OpenAPI, OpenAPIV2, OpenAPIV3 } from 'openapi-types';
-import * as SwaggerSchema from 'swagger-schema-official';
```

Comment:
#Could you please remove this from package.json and add openapi-types to dependencies?

Reply:
Done